data/tlg0527/tlg048/__cts__.xml

```diff
@@ -3,8 +3,8 @@
   <cpt:structured-metadata>
     <dct:isVersionOf>urn:cts:hebrewLit:heb0001.heb010</dct:isVersionOf>
   </cpt:structured-metadata>
-  <ti:translation xml:lang="eng" urn="urn:cts:greekLit:tlg0527.tlg048.opp-grc2" workUrn="urn:cts:greekLit:tlg0527.tlg048">
+  <ti:edition  urn="urn:cts:greekLit:tlg0527.tlg048.opp-grc2" workUrn="urn:cts:greekLit:tlg0527.tlg048">
     <ti:label xml:lang="lat">Isaias</ti:label>
-    <ti:description xml:lang="mul">Septuaginta, Isaias, Ottley, Cambridge, 1904</ti:description>
-  </ti:translation>
-  <ti:translation xml:lang= "eng" urn="urn:cts:greekLit:tlg0527.tlg048.opp-eng2" workUrn="urn:cts:greekLit:tlg0527.tlg048"><ti:label xml:lang="eng">Isaias</ti:label><ti:description xml:lang="mul">Septuaginta, Isaias, Ottley, Cambridge, 1904</ti:description></ti:translation></ti:work>
+    <ti:description xml:lang="mul">Septuaginta in English, Isaias, Ottley, Cambridge, 1904</ti:description>
+  </ti:edition>
+  <ti:edition  urn="urn:cts:greekLit:tlg0527.tlg048.opp-eng2" workUrn="urn:cts:greekLit:tlg0527.tlg048"><ti:label xml:lang="lat">Isaias</ti:label><ti:description xml:lang="mul">Septuaginta, Isaias, Ottley, Cambridge, 1904</ti:description></ti:edition></ti:work>
```

Comment:
I think there is some confusion with too many small details about this file. The file should look like this:
```
<ti:work xmlns:cpt="http://capitains.github.io/xmlns" xmlns:dct="http://purl.org/dc/terms/" xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg0527" xml:lang="grc" urn="urn:cts:greekLit:tlg0527.tlg048">
  <ti:title xml:lang="lat">Isaias</ti:title>
  <cpt:structured-metadata>
    <dct:isVersionOf>urn:cts:hebrewLit:heb0001.heb010</dct:isVersionOf>
  </cpt:structured-metadata>
  <ti:edition  urn="urn:cts:greekLit:tlg0527.tlg048.opp-grc2" workUrn="urn:cts:greekLit:tlg0527.tlg048">
    <ti:label xml:lang="lat">Isaias</ti:label>
    <ti:description xml:lang="mul">Septuaginta, Isaias, Ottley, Cambridge, 1904</ti:description>
  </ti:edition>
  <ti:translation xml:lang= "eng" urn="urn:cts:greekLit:tlg0527.tlg048.opp-eng2" workUrn="urn:cts:greekLit:tlg0527.tlg048">
    <ti:label xml:lang="eng">Isaias</ti:label>
    <ti:description xml:lang="mul">Septuaginta in English, Isaias, Ottley, Cambridge, 1904</ti:description>
  </ti:translation>
</ti:work>
```
One `<ti:work>` node containing the `<ti:title>` of the work and the `<cpt:structured-metadata>`  for it. Then you have two different instantiations of this work, one an edition and one a translation. The edition is the one with the URN ending in `grc2` and the translation is the one ending in `eng2`.
The metadata files can be confusing. If there is still confusion, just let me know and we can meet about it.